velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -1126,5 +1126,28 @@ TEST_F(DateTimeFunctionsTest, timestampToMillis) {
   EXPECT_EQ(timestampToMillis("-292275055-05-16 16:47:04.192"), kMinBigint);
 }
 
+TEST_F(DateTimeFunctionsTest, dateAddYMInterval) {
+  const auto makeInput = [&](const std::string& date, int32_t interval) {
+    return makeRowVector({
+        makeNullableFlatVector<int32_t>({parseDate(date)}, DATE()),
```

Comment:
Why needs to test RowVector? Can we test row by row like this? https://github.com/facebookincubator/velox/pull/12152/files#diff-f7042b580cdd92f6b4ba8b77183933cbb460c88a6deed81f767a35bb4d2cc40eR1114